OnlyOffice documentation for license path (#3281)
owncloud/docs: modules/admin_manual/pages/appliance/configuration/office.adoc

= Working on Documents in the ownCloud Appliance
:toc: right
:page-aliases: appliance/office.adoc

> :onlyoffice-es-install-url: https://helpcenter.onlyoffice.com/installation/docs-enterprise-install-ubuntu.aspx

Reviewer: ```suggestion
:onlyoffice-es-install-url: https://helpcenter.onlyoffice.com/installation/docs-enterprise-activation.aspx
```

* Select on the App name
* Upgrade the App

> == Registering your Enterprise Edition
> 
> If you bought the Enterprise Edition and received the **license.lic** file, you need to import it to unlock the enterprise features.
> 
> When you are a Debian based Linux distributive user, the license.lic file must be placed in the following folder:
> 
> ----
> /var/www/onlyoffice/Data/license.lic
> ----
> 
> Now you will have access to the enterprise features of onlyoffice.
> 
> Additional information can be found https://helpcenter.onlyoffice.com/installation/docs-enterprise-install-ubuntu.aspx[in the onlyoffice documentation]

Reviewer: ```suggestion
== Registering your ONLYOFFICE Enterprise Edition

If you purchased the ONLYOFFICE Enterprise Edition and received the **license.lic** file, you need to import it to unlock the enterprise features.

When you are a Debian based Linux distribution user, the license.lic file must be placed into the following folder:

----
/var/www/onlyoffice/Data/license.lic
----

Doing so, you will have access to the enterprise features of ONLYOFFICE.

Additional information can be found in the https://helpcenter.onlyoffice.com/installation/docs-enterprise-install-ubuntu.aspx[ONLYOFFICE Documentation]
```

Reviewer: ONLYOFFICE is a proper name and needs to be written in capital letters as it is done on their homepage

Author: agreed

* Select on the App name
* Upgrade the App

== Registering your Enterprise Edition

If you bought the Enterprise Edition and received the **license.lic** file, you need to import it to unlock the enterprise features.

When you are a Debian based Linux distributive user, the license.lic file must be placed in the following folder:

----
/var/www/onlyoffice/Data/license.lic
----

Now you will have access to the enterprise features of onlyoffice.

Additional information can be found https://helpcenter.onlyoffice.com/installation/docs-enterprise-install-ubuntu.aspx[in the onlyoffice documentation]

=== Docker

> When you are using the docker container, the license.lic file can be placed to any folder (e.g. /app/onlyoffice/DocumentServer/data/license.lic), but you will need to mount this folder when the container starts:

Reviewer: ```suggestion
When you are using the docker container, the `license.lic` file can be placed to any folder (e.g. /app/onlyoffice/DocumentServer/data/license.lic), but you will need to mount this folder when the container starts:
```

> == Registering your ONLYOFFICE Enterprise Edition
> 
> If you purchased the ONLYOFFICE Enterprise Edition and received the `license.lic` file, you need to import it to unlock the enterprise features.
> 
> When you are a Debian based Linux distribution user, the `license.lic` file must be placed into the following folder:
> 
> ----
> /var/www/onlyoffice/Data/license.lic
> ----
> 
> Doing so, you will have access to the enterprise features of ONLYOFFICE.
> 
> Additional information can be found in the {onlyoffice-es-install-url}[ONLYOFFICE Documentation]
> 
> === Docker
> 
> When you are using the docker container, the `license.lic` file can be placed to any folder (e.g. /app/onlyoffice/DocumentServer/data/license.lic), but you will need to mount this folder when the container starts:
> 
> ----
> docker run -itd -p 80:80 --restart=always \
> -v /app/onlyoffice/DocumentServer/data:/var/www/onlyoffice/Data \
> onlyoffice/documentserver-ie
> ----
> 

Reviewer 2: ```suggestion
== Registering your ONLYOFFICE Enterprise Edition

If you purchased the ONLYOFFICE Enterprise Edition and received the `license.lic` file, you need to import it to unlock the enterprise features.

In Univention appliance, the `/var/lib/univention-appcenter/apps/onlyoffice-ie/Data` directory on the host computer is mounted to the `/var/www/onlyoffice/Data` data volume in the container, so the `license.lic` file can be placed to the following folder on the host computer:
```
/var/lib/univention-appcenter/apps/onlyoffice-ie/Data/license.lic
```
After the file is placed there, your version of ONLYOFFICE Enterprise Edition will become registered and you will have access to the enterprise features of ONLYOFFICE.

Additional information can be found in the {onlyoffice-es-install-url}[ONLYOFFICE Documentation]
```
